pkg/channel/http/http_channel.go

```diff
@@ -49,7 +49,6 @@ func CreateLocalChannel(port, maxConcurrency int, spec config.TracingSpec, sslEn
 	c := &Channel{
 		client: &fasthttp.Client{
 			MaxConnsPerHost:           1000000,
-			ReadTimeout:               channel.DefaultChannelRequestTimeout,
```

Comment:
what's the default readtimeout in fasthttp client ? this will use the default one.. we need to check.

Reply:
It's unlimited.